BackwardsCompatibility.md

```diff
@@ -0,0 +1,29 @@
+# Backward Compatibility
+The current approach to achieving backwards compatibility with EVM1 is to
+support both of the instruction sets with the option to transcompiling EVM1 to 
+EVM2. This approach gives clients optionality when dealing with EVM1 code.
+A client can either implement only a WASM VM and transcompile all of the EVM1
+code. Or a client can implement a wasm VM and EVM1 VM and leave the old code as
+is.
+
+## Gas Prices
+In eWASM we will introduce sub-gas units so that each EVM1 opcode's
```

Comment:
yeah If we can provided a way to do differential metering without added to much complexity then I think that would be ideal.